### PR TITLE
Add configuration to redirect action

### PR DESCRIPTION
**Pull request by nilakhan**

### CODE REVIEW

File: concepts/changelog.md [hunk 1]
```diff
@@ -11,6 +11,46 @@ This changelog covers specific API-level changes in Microsoft Graph v1.0 and bet
...
+
+### Devices and apps | Cloud printing
+
+| **Change type** | **Version**   | **Description**                          |
+|:---|:---|:---|
+| Addition | beta | Added HasPhysicalDevice property to [printer](/graph/api/resources/printer?view=graph-rest-beta) resource. | 
+| Addition | beta | Added [abort](/graph/api/printjob-abort?view=graph-rest-beta) action to [printJob](/graph/api/resources/printjob?view=graph-rest-beta) resource. | 
+| Addition | beta | Added configuration property to [redirect](/graph/api/printjob-redirect?view=graph-rest-beta) action on printJob(/graph/api/resources/printjob?view=graph-rest-beta) resource. | 
+
+### Identity and access
```
Comment (braedenp-msft) [inline concepts/changelog.md:30]: Are non-UP/webhook additions a result of a merge conflict? Can we rebase and get rid of these?

File: concepts/changelog.md [hunk 2]
```diff
@@ -11,6 +11,46 @@ This changelog covers specific API-level changes in Microsoft Graph v1.0 and bet
...
+
+| **Change type** | **Version**   | **Description**                          |
+| :-------------- | :------------ | :--------------------------------------- |
+| Addition | beta | Added [printTaskDefinition](/graph/api/resources/printtaskdefinition?view=graph-rest-beta&preserve-view=true) to the resources supported for [change notifications](/graph/webhooks).|
+
+### Devices and apps | Cloud printing
+
+| **Change type** | **Version**   | **Description**                          |
+|:---|:---|:---|
+| Addition | beta | Added HasPhysicalDevice property to [printer](/graph/api/resources/printer?view=graph-rest-beta) resource. |
```
Comment (braedenp-msft) [inline concepts/changelog.md:26]: Lowercase "h"

File: api-reference/beta/api/printjob-abort.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+```http
+POST /print/printers/{id}/jobs/{id}/abort
+```
+## Request headers
+| Name          | Description   |
+|:--------------|:--------------|
+| Authorization | Bearer {token}. Required. |
+
+## Request body
+In the request body, you can optionally provide the reason why job is being aborted.
```
Comment (braedenp-msft) [inline api-reference/beta/api/printjob-abort.md:40]: Grammar: "the reason why **the** job is being aborted"

**Review by braedenp-msft — APPROVED** [review]

**Comment by nilakhan** [comment]
Duplicate PR

**Pull request closed without merging**